README.md

```diff
@@ -6,34 +6,81 @@
 
 At the core of Dagger.jl is a scheduler heavily inspired by [Dask](http://dask.pydata.org/en/latest/). It can run computations represented as [directed-acyclic-graphs](https://en.wikipedia.org/wiki/Directed_acyclic_graph) (DAGs) efficiently on many Julia worker processes.
 
-# DAG creation interface
 
-Here is an example DAG:
+## Installation
+
+You can install it by typing
+
+```julia
+julia> ] add Dagger
+```
+
+## Usage
+
+Once installed, the `Dagger` package can by used by typing
+
+```julia
+using Dagger
+```
+
+The main function for using Dagger is *delayed*
+
+`delayed(f; options...)`
+
+When called creates a `Thunk` object representing a call to function `f` with the given arguments. If it is called with other thunks as input, then they form a graph with input nodes directed at the output. The function `f` get the result of the input Thunks.
```

Comment:
Would you mind adding a comment that `Thunk`s don't pass keyword argument to the function `f`, and that the `options...` kwargs are passed to the scheduler to control its behavior?

Reply:
One other thing - why was this changed to remove "Returns a function which..."? I'm pretty this is now less accurate.